From 30990438e2a28d203e5bf5bd02a89fb855ed8d4c Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@suse.de>
Date: Mon, 23 Mar 2009 09:33:20 -0700
Subject: [PATCH] Staging: epl: remove NULL from global.h

Don't define something that is already defined in the kernel.

Cc: Daniel Krueger <daniel.krueger@systec-electronic.com>
Cc: Ronald Sieber <Ronald.Sieber@systec-electronic.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/epl/global.h |    3 ---
 1 file changed, 3 deletions(-)

--- a/drivers/staging/epl/global.h
+++ b/drivers/staging/epl/global.h
@@ -222,9 +222,6 @@
 #ifndef FALSE
 #define FALSE 0x00
 #endif
-#ifndef NULL
-#define NULL ((void *) 0)
-#endif
 #ifndef _TIME_OF_DAY_DEFINED_
 typedef struct {
 	unsigned long int m_dwMs;
